server/migrations/20230523163856-remove-OtherUnexpectedBehaviorText.js

```diff
@@ -0,0 +1,163 @@
+                        p <
+                        testPlanRunData[i].testResults[j].scenarioResults
+                            .length;
+                        p += 1
+                    ) {
+                        if (
+                            !testPlanRunData[i].testResults[j].scenarioResults[
+                                p
+                            ].unexpectedBehaviors
+                        ) {
+                            continue;
+                        }
+
+                        if (
+                            Array.isArray(
+                                testPlanRunData[i].testResults[j]
+                                    .scenarioResults[p].unexpectedBehaviors
+                            ) &&
+                            testPlanRunData[i].testResults[j].scenarioResults[p]
+                                .unexpectedBehaviors.length < 1
+                        ) {
+                            continue;
+                        }
+                        for (
+                            let s = 0;
+                            s <
+                            testPlanRunData[i].testResults[j].scenarioResults[p]
+                                .unexpectedBehaviors.length;
+                            s += 1
+                        ) {
+                            const unexpectedBehavior =
+                                testPlanRunData[i].testResults[j]
+                                    .scenarioResults[p].unexpectedBehaviors[s];
+                            if (
+                                unexpectedBehavior.id !== 'OTHER' &&
+                                unexpectedBehavior.otherUnexpectedBehaviorText ===
+                                    null
+                            ) {
+                                delete unexpectedBehavior.otherUnexpectedBehaviorText;
+
+                                updateParams.testResults[j] =
+                                    testPlanRunData[i].testResults[j];
+
+                                needsUpdate = true;
+                            }
+                        }
+                    }
+                }
+
+                if (needsUpdate)
+                    await updateTestPlanRun(testPlanRunId, updateParams);
+            }
+
+            for (let i = 0; i < testPlanReportsData.length; i++) {
+                const testPlanReportId = testPlanReportsData[i].id;
+                const status = testPlanReportsData[i].status;
+                if (status === 'DRAFT') {
+                    let updateParams = {};
+                    const { testPlanReport } = await populateData(
+                        {
+                            testPlanReportId
+                        },
+                        {}
+                    );
+
+                    const conflicts = await conflictsResolver(
+                        testPlanReport,
+                        null,
+                        { atLoader, browserLoader }
```

Comment:
Just a thought, and not suggesting as something to act on.

Right now, the GQL context could be `{ user, atLoader, browserLoader }` with the change in #623. Now that I've seen its usage a bit more, it seems `atLoader` and `browserLoader` being present is a must in most queries.

So I'd think that when `context` is null in the resolvers it's called, it may be preferred if it simply defaults to `{ atLoader, browserLoader }`. I'm unsure if sure if there's any implications to that though, and also acknowledge that it would a considerable effort in determining that. But creating the objects here to solely retrieve the conflict results here felt out of place.